src/components/avatar-image.tsx

```diff
@@ -24,13 +21,11 @@ export const AvatarImage: React.VoidFunctionComponent<AvatarImageProps> = ({
   size,
 }) => {
   return (
-    <ImageContainer>
-      <StyledGravatar
-        email={email || ""}
-        default="mp"
-        rating="g"
-        size={size || defaultAvatarSizePix}
-      />
-    </ImageContainer>
+    <StyledGravatar
```

Comment:
Prevents avatar shrinking
<img width="401" alt="Screenshot 2022-01-30 at 00 16 50" src="https://user-images.githubusercontent.com/608862/151681938-84e1bba5-d315-46f4-986d-7eaf298b0c2c.png">


Related to #31